nix/hydra/shell.nix

```diff
@@ -145,4 +158,5 @@ in
   cabalOnly = cabalShell;
   exes = exeShell;
   demo = demoShell;
+  ci = ciShell;
```

Comment:
I thought we have this already, but in fact we _had_ this already: https://github.com/cardano-scaling/hydra/commit/7a8ba8e712fdbcb32c9a77f761673e3067dc3258